app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    like = Like.new
+    like.author = current_user
+    like.post = post
```

Comment:
- [ ] [OPTIONAL] You have implemented the like functionality correctly, however I would like to show you a way you can make use of the relationships between the user and the likes to make your code even shorter and more readable.
Remember that there is a one to many relationship between a user and the likes, right? Now you can do something like this 👇 
``` ruby
like = current_user.likes.new(post_id: post.id)
like.save
```

Reply:
Thanks, @codepantha for these suggestions. I will surely adopt these changes👍